py2exe/hooks.py

```diff
@@ -397,20 +401,80 @@ def hook_matplotlib(finder, module):
     # matplotlib <=3.3.4 requires '_get_data_path' to be patched
     # matplotlib >= 3.4.0 requires 'get_data_path' to be patched
     mpl_version = pkgversion.parse(matplotlib.__version__)
-    node_to_be_patched = 'get_data_path' if mpl_version >= pkgversion.parse('3.4.0') else '_get_data_path'
+    get_data_node_to_be_patched = 'get_data_path' if mpl_version >= pkgversion.parse('3.4.0') else '_get_data_path'
+
+    # matplotlib >= 3.7.0 requires patching '_delvewheel_init_patch_1_3_3'
+    if mpl_version >= pkgversion.parse('3.7.0'):
+
+        tree = ast.parse(module.__source__)
+        devel_node_to_be_patched = '_delvewheel_init_patch_1_3_3'
+
+        mpl_libs_path = os.path.abspath(os.path.join(os.path.dirname(matplotlib.__file__), os.pardir, 'matplotlib.libs'))
+        if os.path.isdir(mpl_libs_path):
+            from os import listdir
```

Comment:
`os` is already imported in `hooks`. Please remove this and just call `os.listdir` when needed.